source/docs/hardware-reference/talonfx/index.rst

```diff
@@ -64,20 +64,210 @@ Status Light Reference
-| Green/Orange            | Device is in bootloader.                                                                                 |
-+-------------------------+----------------------------------------------------------------------------------------------------------+
+.. raw:: html
+
+    <style>
+        .led {
+            float: left;
+            height: 20px;
+            width: 20px;
+            border: 1px solid black;
+            border-radius: 10px;
+            margin: 5px;
+            background-color: darkgray;
+        }
+        .ledGroup {
+            display: inline-block;
+            height: 20px;
+        }
+        table.center, table.center th, table.center td {
+            border: 1px solid white;
+            border-collapse: collapse;
+            padding: 5px;
+            text-align: center;
+        }
+
+        .tableOverflow {
+            overflow: scroll;
+        }
+
+        td.overflow {
+            max-width: 550px;
+            overflow: scroll;
+        }
+
+        @media screen and (max-width: 480px) {
+            td.overflow {
+                max-width: 0;
+                overflow: scroll;
+            }
+
+            .tableOverflow {
+                max-width: 480px;
+            }
+        }
+    </style>
+
+    <div class="tableOverflow">
+        <table class="center">
+            <tr>
+                <th colspan="3">Blink Codes</th>
+            </tr>
+            <tr>
+                <th colspan="3">Disabled Codes</th>
+            </tr>
+            <tr>
+                <th>Animation (Click to animate)</th>
+                <th>LED State</th>
+                <th>Meaning</th>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='0' offtime='0' oncolor='black' offcolor='black'></div>
+                                          <div class='led' ontime='0' offtime='0' oncolor='black' offcolor='black'></div></div></td>
+                <td>LEDs Off</td>
+                <td>No Power</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='red' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='black' offcolor='red'></div></div></td>
+                <td>Blinking Alternating Red</td>
+                <td>Talon FX does not have valid CAN/PWM signal.</td>
+            </tr>
+            <tr>
+                <td><div class='ledGroup'><div class='led' ontime='300' offtime='300' oncolor='orange' offcolor='black'></div>
+                                          <div class='led' ontime='300' offtime='300' oncolor='black' offcolor='orange'></div></div></td>
+                <td>Blinking Alternating Orange</td>
+                <td>Talon FX has valid CAN/PWM signal. If on CAN, Phoenix is not running in robot controller <b>or</b> Talon FX does not have good CAN connection to robot controller.</td>
```

Comment:
I would like the descriptions for the same problem to match between Pigeon 2 and TalonFX. I like this more than the Pigeon description at least

Reply:
The descriptions look the same to me in WinMerge though, other than the first sentence (which is different because TalonFX supports PWM).